WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Package Details/ShippingLabelPackageDetailsViewModel.swift

```diff
@@ -132,6 +189,19 @@ private extension ShippingLabelPackageDetailsViewModel {
         }
         stores.dispatch(action)
     }
+
+    func syncPackageDetails() {
+        let action = ShippingLabelAction.packagesDetails(siteID: order.siteID) { [weak self] result in
+            switch result {
+            case .success:
+                self?.packagesResponse = try? result.get()
+            case .failure:
+                DDLogError("⛔️ Error synchronizing package details")
```

Comment:
Probably good to output the error here to have meaningful logs!

Reply:
Do you mean in the console? Because the error should already be printed on the console